### PR TITLE
DOCSP-37005 - Periodic Executors

### PR DESCRIPTION
**Pull request by mongoKart**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-37005
Staging - https://preview-mongodbmongokart.gatsbyjs.io/pymongo/docsp-37005-periodic-executors/fundamentals/periodic-executors/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals.txt [hunk 1]
```diff
@@ -2,4 +2,17 @@
 
 ============
 Fundamentals
-============
\ No newline at end of file
+============
+
+.. meta::
+   :description: Learn how to use the (+driver-short+} to execute operations on MongoDB.
```
Comment (jordan-smith721) [inline source/fundamentals.txt:8]:
```suggestion
   :description: Learn how to use (+driver-short+} to execute operations on MongoDB.
```

File: source/fundamentals/periodic-executors.txt [hunk 2]
```diff
@@ -65,52 +76,42 @@ callback to terminate itself soon after the topology is freed.
...
 
 We erred on the side of prompt cleanup, and set the check interval at 100ms. We
 assumed that checking a flag and going back to sleep 10 times a second was
 cheap on modern machines.
 
-Starting in Python 3.2, the builtin C implementation of lock.acquire takes a
-timeout parameter, so Python 3.2+ Condition variables sleep simply by calling
-lock.acquire; they are implemented as efficiently as expected.
+Starting in Python 3.2, the built-in C implementation of the ``lock.acquire()`` method
+takes a ``timeout`` parameter, so Python 3.2+ Condition variables sleep simply by calling
```
Comment (jordan-smith721) [inline source/fundamentals/periodic-executors.txt:102]: Not sure if Condition should be lowercased, or if it's a keyword that should be monospaced?

```suggestion
takes a ``timeout`` parameter, so Python 3.2+ condition variables sleep simply by calling
```

**Review by jordan-smith721 — APPROVED** [review]
LGTM w/ a couple small things

File: source/fundamentals/periodic-executors.txt (hunk 2 above)
Comment (mongoKart) [reply source/fundamentals/periodic-executors.txt:102]: yeah, should be lowercase

**Pull request merged**